corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -873,6 +875,21 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
             this.labelWidth = constants.LABEL_WIDTH;
             this.questionTileWidth = constants.FULL_WIDTH;
         }
+
+        function setCustomInputWidth() {
+            const shortWidth = new RegExp(constants.SHORT_WIDTH);
+            const mediumWidth = new RegExp(constants.MEDIUM_WIDTH);
+            $(document).ready(function () {
+                const entryWidgetContainer = $(`#${self.entry.entryId}`).closest('.widget-container');
```

Comment:
Does this target the same element as the element where `controlWidth` is applied? https://github.com/dimagi/commcare-hq/blob/946e797a1dc88d44cff2112da5189fdeae9b2004/corehq/apps/cloudcare/templates/form_entry/templates.html#L257

Could the class assignment be done via knockout similar to how it's done for `perRowPattern` instead of removing then adding a class in?

Reply:
That's possible. The thing I like about how it is here is that it consolidates a lot of the width logic in the `Question.prototype.setWidths` method.